sass/components/_emoticons.scss

```diff
@@ -81,6 +81,10 @@
     width: 315px;
     z-index: 999;
 
+    .emoji-picker__header{
+        background: var(--sidebar-header-bg);
+    }
```

Comment:
@jgilliam17, this applies the background colour of the emoji picker header in mobile view.